docs/en/sql-reference/sql-functions/JAVA_UDF.md

```diff
@@ -402,6 +402,7 @@ PROPERTIES (
 | symbol    | The name of the class for the Maven project to which the UDF belongs. The value of this parameter is in the `<package_name>.<class_name>` format. |
 | type      | The type of the UDF. Set the value to `StarrocksJar`, which specifies that the UDF is a Java-based function. |
 | file      | The HTTP URL from which you can download the JAR file that contains the code for the UDF. The value of this parameter is in the `http://<http_server_ip>:<http_server_port>/<jar_package_name>` format. |
+| isolation | (Optional) To share function instances across UDF executions and support static variables, set this to "shared". |
```

Comment:
Can you explain the default value of this property and all the optional values?

Reply:
For the key "isolation", [the only accepted value](https://github.com/digitalpoetry/starrocks/blob/0123b3d88e6cc002eba2baf96d2e64be5a63c109/fe/fe-core/src/main/java/com/starrocks/sql/analyzer/CreateFunctionAnalyzer.java#L548) here is "shared".
All other values are no-ops.